deploy/charts/klicker-uzh-v2/templates/cm-backend-assessment.yaml

```diff
@@ -13,38 +13,14 @@ data:
   APP_MANAGE_SUBDOMAIN: {{ .Values.assessment.backendGraphql.appManageSubdomain | quote }}
```

Comment:
if we merge this as is while still having deployments online on the old infra, everything will break -> make a new copy of the "chart" (klicker-uzh-v3) and dont change anything on the klicker-uzh-v2 chart. this will allow us to cleanly upgrade and it also aligns the version of the chart with the actual v3 klicker. so overall, this PR should not make any changes to the v2 chart, and just create and reference a new v3 chart everywhere.